API-Automation/tasks/api-tasks.js

```diff
@@ -0,0 +1,307 @@
+import axiosInstance from "../support/axiosConfig.js";
+import {
+  createUserPath,
+  createUserWithListData,
+  createUserWithListPath,
+  deleteUserInvalidUsernamePath,
+  getUserPath,
+  newUserData,
+  updateUserData,
+  userLoginPath,
+  userLoginPathUsername,
+  deleteUserPath,
+  userLogoutPath,
+  addNewPetPath,
+  newPetData,
+  editPetPath,
+  editPetData,
+  findByStatusPath,
+  findPetByIdPath,
+  deletePetPath,
+  createOrderPath,
+  createOrderData,
+  getPurchaseByPetIdPath,
+  getInventoryPath,
+  deleteOrderByIdPath
+} from "../data/commonData.js";
+
+export async function getUserByUsername() {
+  try {
+    const response = await axiosInstance.get(getUserPath.path);
+    return {
+      status: response.status,
+      data: response.data
+    };
+  } catch (error) {
+    return {
+      error: error.response.status,
+      error: error.message
+    };
```

Comment:
Let's return here the whole response which client returns, and then in tests itself we can manipulate with body/headers/statuses etc.